examples/experimental/onnx/classification/onnx_ptq_classification.py

```diff
@@ -89,8 +75,6 @@ def run(onnx_model_path: str, output_model_path: str,
         "--batch_size", help="Batch size for initialization", type=int, default=1)
     parser.add_argument(
         "--shuffle", help="Whether to shuffle dataset for initialization", default=True)
-    parser.add_argument(
-        "--evaluate", help="Run an evaluation step for the final quantized model", action="store_true")
```

Comment:
Let's keep the option in the sample but remove this functionality from the quantization API.

Reply:
In this case, it is necessary to implement the evaluation method for the sample, right?